src/django_components_js/src/index.ts

```diff
@@ -0,0 +1,13 @@
+import { createComponentsManager } from './manager';
```

Comment:
This file defines the API of the JS code. E.g. individual manager functions would be called like this in the browser:

```js
Components.manager.callComponent(...);
```

NOTE: This code is NOT meant to be used by end-users.